examples/crm/src/contacts/ContactList.tsx

```diff
@@ -38,6 +40,9 @@ export const ContactList = () => {
 const ContactListActions = () => (
     <TopToolbar>
         <SortButton fields={['last_name', 'first_name', 'last_seen']} />
+        <ImportButton
+            sampleUrl={toDataUrl(sampleCsv.default, 'text/csv;charset=utf-8')}
```

Comment:
I wouldn't put it here as an argument, and import the sampleCSV directly in ImportButton.